packages/angular-ui-components/README.md

```diff
@@ -0,0 +1,27 @@
+# AngularUIComponents
```

Comment:
This should be updated with useful information like: How to use, ...
Please add a TODO comment.  So that we don't forget